commerce-theme-speedwell/commerce-theme-speedwell/src/js/main.js

```diff
@@ -1,12 +1,9 @@
 AUI().ready(
 	function() {
-		console.log('%c[ AUI ready ]', 'background-color: #000; color: #00FFFF');
-
-		if (!!Speedwell && !!Speedwell.features) {
+		if (!!Speedwell && !!Speedwell.features && !!Speedwell.features.init) {
 			Speedwell.features.init.initializeFeatures();
 
 			Speedwell.features.sliders = [];
-			Speedwell.features.sliderCallbacks = [];
```

Comment:
Why was this removed? It is necessary for every main.js reload to clean the queue of callbacks, if there are any. If not, the following 'if' won't trigger.

Reply:
Ok. Yeah when it runs here it clears the queue right before checking whether there's anything in it. Makes sense to me, if we need to clean it, to clean it after it's used. Don't think I can make the change tonight - feel free to send a new pull with the correct changes tomorrow.